package.json

```diff
@@ -11,7 +11,7 @@
     },
     "icon": "media/bazel-logo.png",
     "engines": {
-        "vscode": "^1.27.0"
+        "vscode": "^1.30.0"
```

Comment:
Is this change necessary for any of the functionality in this PR? If not, please revert it and also revert the changes to package-lock.json.

Reply:
A colleague pointed out that 1.30 is when the clipboard APIs were introduced. I've pushed a separate change to update the version requirement (#114); can you revert these package(-lock).json changes and rebase so this will be a standalone change?

Reply:
Yeah, that's why I changed the min-version, thanks for your change.